api/pkg/apis/v1alpha1/providers/stage/create/create.go

```diff
@@ -139,12 +140,12 @@ func (i *CreateStageProvider) Process(ctx context.Context, mgrContext contexts.M
 			objectNamespace = "default"
 		}
 
-		if action == "remove" {
+		if strings.EqualFold(action, "remove") {
```

Comment:
do we want to compare with the constant here as well?

Reply:
This is passed in from user's yaml file. We cannot guarantee it's well formatted.

Reply:
I mean to use strings.EqualFold(action, <the constant>)

Reply:
Fixed.